docs/code_style.md

```diff
@@ -30,3 +30,8 @@ The quickest way to correct import style locally is to let isort make the change
 Note: It's not possible to disable isort wrapping style checking, so for now we've chosen the
 most deterministic [wrapping mode](https://github.com/timothycrosley/isort#multi-line-output-modes)
 to reduce the line length guess-work when adding imports, even though it's not the most concise.
+
+UI
+--------------
+
+We use the [Airbnb](https://github.com/airbnb/javascript) style guide for Javascript and validate it with eslint (see Validating Javascript in the [UI development section](ui/installation.md#validating-javaScript). For CSS, we use Bootstrap and [reactstrap](https://reactstrap.github.io/). We use Bootstrap's utility classes as much as possible before adding custom CSS to a style sheet. Any custom style that can be made reusable should be named generically and stored in the ``ui/css/treeherder-global.css`` file.
```

Comment:
For CSS longer term I'd like us to move away from plain bootstrap and use solely Reactstrap (since otherwise we have duplication in the bundle). Perhaps the wording can be tweaked to hint at this?

s/eslint/ESLint/

Reply:
Sure, will do.